CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## [Unreleased](https://github.com/openfga/dotnet-sdk/compare/v0.8.0...HEAD)
 
+### Added
+- feat: add server-side `BatchCheck()` method using `/batch-check` API endpoint
```

Comment:
Not sure if the changes from [this comment](https://github.com/openfga/dotnet-sdk/pull/150#discussion_r2503431123) got reverted somehow but I think we can condense this entry a little.

imo, the cross-framework isn't necessary (it should just work) and the behaviour notes maybe fit better on the docs in the readme